var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -62,6 +62,17 @@ class Fms(CMakePackage):
         when="@2023.03",
     )
 
+    # macos needs shared libraries for version 2024.02
```

Comment:
This is a little confusing, because the sharedlibs variant (which should be called shared for consistency with other spack packages) applies to all versions 2024.02+, the only line that is for 2024.02 specifically is 74.

Reply:
I resolved this issue.